pkgs/tools/text/mdbook-pdf/default.nix

```diff
@@ -28,6 +30,12 @@ rustPlatform.buildRustPackage rec {
     CoreServices
   ];
 
+  # Stop downloading from the Internet to
+  # generate the Chrome Devtools Protocol
+  preBuild = ''
+    export DOCS_RS=true
```

Comment:
Shall we set it as `env.DOCS_RS = true;` (aka. structured attrs)

Reply:
Done! Thanks for reviewing!